modules/jwt-cpp/0.7.0.bcr.1/source.json

```diff
@@ -0,0 +1,17 @@
+{
+    "url": "https://github.com/Thalhammer/jwt-cpp/archive/refs/tags/v0.7.0.tar.gz",
+    "integrity": "sha256-uesnDjuoIh5LK8OHI8mhy0+mwkGkKQi5ozTa/zETdAY=",
+    "strip_prefix": "jwt-cpp-0.7.0",
+    "patch_strip": 1,
+    "patches": {
+        "add-build.patch": "sha256-893niHcbMlUl7JFb82Wr3tmirBTWi5pSUQQR64pnTIo=",
```

Comment:
For the patches that don't change from `0.7.0`, you can just use the relative path here, like "add-build.patch": ../0.7.0/patches/add-build.patch" except for the patch that actually changes

Reply:
Sorry, it's instead like this: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/osqp-cpp/0.0.0-20231004-4343373.bcr.1/source.json#L6

Reply:
Thanks, I adapted it.